src/components/Login/SubmitButton.js

```diff
@@ -1,12 +1,12 @@
 import React from "react";
 function Submit(props) {
-  console.log(props.class);
   return (
-    <input
+    <button
```

Comment:
why you are not using mui button

Reply:
when i am finishing the project , in final touches i will convert it to mui button : ).